lldb/include/lldb/Utility/LLDBLog.h

```diff
@@ -49,6 +49,7 @@ enum class LLDBLog : Log::MaskType {
   Watchpoints = Log::ChannelFlag<30>,
   OnDemand = Log::ChannelFlag<31>,
   Source = Log::ChannelFlag<32>,
+  AlwaysOn = Log::ChannelFlag<32>,
```

Comment:
does this number need to be unique?

Reply:
Yes, definitely. But this code shouldn't have been part of this PR (I created a new category like I did on Phab). Let me clean this up.